docs/docs/refs-and-the-dom.md

```diff
@@ -0,0 +1,88 @@
+---
+id: refs-and-the-dom
+title: Refs and the DOM
+permalink: docs/refs-and-the-dom.html
+---
+
+In the typical React dataflow, `props` are the only way that parent components interact with their children. To modify a child, you re-render it with new `props`. However, there are a few cases where you need to imperatively modify a child outside of the typical dataflow. The child to be modified could be an instance of a React component, or it could be a DOM element. For both of these cases, React provides an escape hatch.
```

Comment:
Nit: can unbacktick `props`? We normally use them as a word in sentences. We can link them to "components and props" guide.